app/scripts/modules/core/src/managed/constraints/AllowedTimes.less

```diff
@@ -0,0 +1,4 @@
+.DeploymentWindow {
+  margin-top: 4px;
+  padding-left: 16px;
+}
```

Comment:
You can remove this file and use the classnames `sp-margin-xs-top sp-padding-l-left` instead.

See the [styleguide reference](https://github.com/spinnaker/styleguide/blob/master/src/styles/spacing.css) for more info.

Reply:
I was wondering about the source of these classes! thanks
btw, feels like we could use [tailwindcss](https://tailwindcss.com/). It's really awesome

Reply:
Fixed, thanks!